### PR TITLE
4.3 cluster etcd operator: status update changes

### PR DESCRIPTION
**Pull request by alaypatel07**
/hold

this is not tested, my libvirt vm gets low on resources once the bootstrap member is removed. Probably need aws to test this

### CODE REVIEW

File: pkg/operator/configobservation/etcd/observe_etcd.go [hunk 1]
```diff
@@ -139,7 +139,7 @@ func ObserveClusterMembers(genericListers configobserver.Listers, recorder event
 func ObservePendingClusterMembers(genericListers configobserver.Listers, recorder events.Recorder, currentConfig map[string]interface{}) (observedConfig map[string]interface{}, errs []error) {
 	listers := genericListers.(configobservation.Listers)
 	observedConfig = map[string]interface{}{}
-	clusterMemberPath := []string{"cluster", "pending"}
+	clusterMemberPath := []string{"cluster", "members"}
```
Comment (hexfusion) [inline pkg/operator/configobservation/etcd/observe_etcd.go]: what is the theory here?

Comment (alaypatel07) [reply pkg/operator/configobservation/etcd/observe_etcd.go]: removed the commit. Thanks for the catch

**Review by hexfusion — COMMENTED** [review]
lgtm, thanks @alaypatel07

**Pull request merged**